table.go

```diff
@@ -0,0 +1,49 @@
+package authenticated_block_store
+
+import (
+	"fmt"
+)
+
+type TableManager struct{
+	size int
+}
+// Table API, called by: package.api()
```

Comment:
I think this comment confuses me more than it helps. Suggest removing, or change to something like "Table constructor".